### ISSUE TITLE
Fail to validate a call to a keyword when more than 2 return variables are on their own line

### ISSUE THREAD

**Issue by yann-mk** [nokia/RED#302]
When a keyword returns 2 or more values, and the call is split over 2 lines, then RED's validation displays a wrong error.

See attached screenshots: 
* validation-works-one-line.png  line 151 keyword call on one line, no error as expected
* validation-fails-two-lines.png  line 151 and 152  keyword call on two lines where the first lines contains only the variables to assign, RED shows an error, but there is no error (Robot script works correctly in that case).

![validation-works-one-line](https://user-images.githubusercontent.com/49915465/58079298-f23c2800-7bb0-11e9-9b3e-697fe5df8b82.png)

![validation-fails-two-lines](https://user-images.githubusercontent.com/49915465/58079284-ebadb080-7bb0-11e9-9d43-9eac094b13a4.png)

Here is the code from the screenshots:

```robotframework
Get EMSG Box Data
    [Documentation]    Return the payload of EMSG box as a list of bytes.
    [Arguments]    ${emsg_content}

    ${emsg_content_size}=    BuiltIn.Get Length    ${emsg_content}
    @{emsg_content}=    Collections.Convert To List    ${emsg_content}
    # EMSG box header is a FullBox that includes a 4-byte extension (see 5.10.3.3.3 of c065274_ISO_IEC_23009-1_2014.pdf)
    @{box_header}=    Collections.Get Slice From List    ${emsg_content}    0    12
    ${declared_size}    ${box_type}    ${full_box_extension}=
    ...    BuiltIn.Evaluate    struct.unpack('>I4sI', ''.join(${box_header}))    modules=struct
    BuiltIn.Should Be Equal As Strings    ${box_type}    emsg
    ...    Was expecting type 'emsg' in header of EMSG box but got '${box_type}' instead.
    BuiltIn.Should Be Equal As Integers    ${emsg_content_size}    ${declared_size}
    ...    The EMSG MP4 box declares a size of ${declared_size} bytes, but the EMSG box is ${emsg_content_size} bytes long.

    ${box_data}=    Collections.Get Slice From List    ${emsg_content}    12

    [Return]    ${box_data}
```

**Comment by joanna-may** [comment]
Hi,
there is problem indeed. The line starting with 
```robotframework
${declared_size}    ${box_type}    ${full_box_extension}=
```
is treated as continuation of line :
```robotframework
@{box_header}=    Collections.Get Slice From List    ${emsg_content}    0    12
```

**Comment by michalanglart** [comment]
Fixed in 0.9.0
